data_validation/cli_tools.py

```diff
@@ -537,6 +537,11 @@ def _configure_schema_parser(schema_parser):
 def _configure_custom_query_parser(custom_query_parser):
     """Configure arguments to run custom-query validations."""
     _add_common_arguments(custom_query_parser)
+    custom_query_parser.add_argument(
```

Comment:
We should make this a required argument with argparse `required=True`

Reply:
Done